cmd/minikube/cmd/start.go

```diff
@@ -359,6 +370,22 @@ func provisionWithDriver(cmd *cobra.Command, ds registry.DriverState, existing *
 	}, nil
 }
 
+func validateKicBaseVersion(r command.Runner) {
+	res, err := r.RunCmd(exec.Command("cat", "/version.json"))
+	if err != nil {
+		out.WarningT("Unable to open version.json: {{.error}}", out.V{"error": err})
```

Comment:
If this command fails we should return as the rest of the them are for sure going to fail at this point.